### ISSUE TITLE
[FEATURE] disable clear search icon

### ISSUE THREAD

**Issue by vlio20** [bithost-gmbh/ngx-mat-select-search#289]
Would be great if the visibility of the search clear button would be configurable .
Something like, showClearSearch

**Comment by vlio20** [comment]
A workaround is to have something like that:
```
<ngx-mat-select-search>
          <span ngxMatSelectSearchClear></span>
        </ngx-mat-select-search>
```

**Comment by macjohnny** [comment]
sounds good, do you want to file a PR with an option for this? please also update the readme

**Comment by vlio20** [comment]
Sure.

**Comment by vlio20** [comment]
checkout https://github.com/bithost-gmbh/ngx-mat-select-search/pull/290

**Comment by macjohnny** [comment]
@vlio20 thanks for your contribution, it was released in version 3.2.0
